app/src/main/java/com/nutomic/syncthingandroid/activities/FirstStartActivity.java

```diff
@@ -389,6 +402,20 @@ private void requestLocationPermission() {
                 Constants.PermissionRequestType.LOCATION.ordinal());
     }
 
+    private void requestNotificationPermission() {
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+            if (ActivityCompat.checkSelfPermission(this, Manifest.permission.POST_NOTIFICATIONS) != PackageManager.PERMISSION_GRANTED) {
+
+                requestPermissions(new String[]{Manifest.permission.POST_NOTIFICATIONS}, 1);
+
+            } else {
```

Comment:
I don't see a point in keeping an empty else block.

Reply:
fixed